core/templates/css/oppia.css

```diff
@@ -413,14 +413,6 @@ legend {
     transition: transform 200ms linear;
     will-change: transform;
   }
-
-  .headroom--pinned {
-    transform: translateY(0%);
-  }
-
-  .headroom--unpinned {
-    transform: translateY(-100%);
-  }
```

Comment:
Can you make sure these classes aren't used elsewhere?

Reply:
I checked and they are begin used but they have other properties with their seperate transform values

Reply:
> Can you make sure these classes aren't used elsewhere?

I checked and they are begin used but they have other properties with their seperate transform values